src/ostree/ostree-trivial-httpd.c

```diff
@@ -511,6 +524,13 @@ run (int argc, char **argv, GCancellable *cancellable, GError **error)
       goto out;
     }
 
+  if (!(opt_random_408s_percentage >= 0 && opt_random_408s_percentage <= 99))
```

Comment:
Minor: the way I like to write these range type checks usually is:

```c
if (!(0 <= opt_random_408s_percentage && opt_random_408s_percentage <= 99))
```

Reply:
Ahh that's OK, I see you're just following the existing formatting of the 500s one.